parm/stage/ocean.yaml.j2

```
@@ -3,7 +3,7 @@ ocean:
         - "{{ COMOUT_OCEAN_RESTART_PREV }}"
     link_req:
         - ["{{ ICSDIR }}/{{ COMOUT_OCEAN_RESTART_PREV | relpath(ROTDIR) }}/{{ m_prefix }}.MOM.res.nc", "{{ COMOUT_OCEAN_RESTART_PREV }}"]
-        {% if OCNRES == "025" %}
+        {% if OCNRES == "025" and not MOM6_INTERP_ICS %}
```

This should always be set, so just remove
```suggestion
        {% if OCNRES == "025" %}
```

---

This is needed. MOM6 mx025 GLORE IC is only 1 file. The yaml tries to grab multiple files.

---

Apologies. I misinterpreted the intent here. I thought `MOM6_INTERP_ICS` was a path, not a boolean.

I'll resolve this and think a bit more. I would prefer the `JGLOBAL_STAGE_IC` not to source `config.ocn` since the ocean config file will not be populated by all experiments. I will come back with a better suggestion tomorrow.